sdk/core/core-amqp/review/core-amqp.api.md

```diff
@@ -368,7 +368,7 @@ export function createSasTokenProvider(data: {
 // @public
 export const defaultCancellableLock: CancellableAsyncLock;
 
-// @public
+// @public @deprecated
```

Comment:
Need not be deprecated anymore, right?

Because this supports the `value` argument and the core-utils one doesn't.